renderer/index.js

```diff
@@ -254,10 +272,7 @@ function updateDockIcon () {
   if (!activeTorrentsExist || progress === 1) {
     progress = -1
   }
-  if (progress !== state.dock.progress) {
-    state.dock.progress = progress
-    electron.ipcRenderer.send('setProgress', progress)
-  }
+  state.dock.progress = progress
```

Comment:
I actually think this is okay because it's just our `state` object so `state.dock` will always exist. The `if (state.dock)` check exists somewhere else.